### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by mateusrodriguesxyz**
Add Swift Package Manager installation guide

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -262,9 +263,7 @@ $ pod install
 
 ### Carthage
 
-[Carthage](https://github.com/Carthage/Carthage) is a decentralized dependency manager that builds your dependencies and provides you with binary frameworks.
-
-You can install Carthage with [Homebrew](http://brew.sh/) using the following command:
+The [Swift Package Manager](https://swift.org/package-manager/) is a tool for managing the distribution of Swift code. It’s integrated with the Swift build system to automate the process of downloading, compiling, and linking dependencies.
```
Comment (AvdLee) [inline README.md]: I think that you've updated the Carthage instructions here with SPM

**Review by AvdLee — CHANGES_REQUESTED** [review]
Thank you so much for this! I'm surprised we didn't have this in the first place. Glad we can improve it this way!

**Comment by ghost** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Messages</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:book:</td>
      <td>

  View more details on <a href="https://app.bitrise.io/build/f6eb36d218e8bc81" target="_blank">Bitrise</a>
  </td>
    </tr>
  
<tr>
      <td>:book:</td>
      <td>Mocker: Executed 18 tests, with 0 failures (0 unexpected) in 1.799 (1.809) seconds</td>
    </tr>
  </tbody>
</table>


<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/swift">Danger Swift</a> against fc842bb1afb7c0379af1a63bc2b691c07185ac89
</p>

File: README.md (hunk 1 above)
Comment (mateusrodriguesxyz) [reply README.md]: Sorry! I've fixed it.

**Review by AvdLee — APPROVED** [review]

**Comment by ghost** [comment]
Congratulations! :tada: This was released as part of [Release 2.2.0](https://github.com/WeTransfer/Mocker/releases/tag/2.2.0) :rocket:<p align="right"><i>Generated by <a href="https://github.com/WeTransfer/GitBuddy">GitBuddy</a></i></p>

**Pull request merged**